Godeps/Godeps.json

```diff
@@ -34,8 +33,8 @@
 		},
 		{
 			"ImportPath": "github.com/opencontainers/runtime-spec/specs-go",
-			"Comment": "v1.0.0-rc1-31-gbb6925e",
-			"Rev": "bb6925ea99f0e366a3f7d1c975f6577475ca25f0"
+			"Comment": "v1.0.0-rc5-30-g3adac26",
```

Comment:
> … what method can you only be updated to 1.0.0-rc5?

In whichever `GOPATH` you use, go to `src/github.com/opencontainers/runtime-spec`.  Then:

    $ git checkout v1.0.0-rc5

Now, back in runtime-tools:

    $ godep update github.com/opencontainers/runtime-spec/specs-go

and you should be good to go.